src/HonzaBotner/appsettings.json

```diff
@@ -3,8 +3,12 @@
         "LogLevel": {
             "Default": "Warning",
             "Microsoft": "Warning",
-            "Microsoft.Hosting.Lifetime": "Information"
+            "Microsoft.Hosting.Lifetime": "Information",
+            "Hangfire": "Information"
         }
     },
+    "DashboardUsers": {
+        "admin": "heslo"
```

Comment:
Very secure :D

Reply:
`git blame` :kappa:

Reply:
Can we move this to heroku secret config?